jsapp/js/components/reports.es6

```diff
@@ -526,6 +526,10 @@ class ReportContents extends React.Component {
         {
           reportData.map((rowContent, i)=>{
             var label = (rowContent.row.label && rowContent.row.label[tnslIndex]) ? rowContent.row.label[tnslIndex] : t('Unlabeled');
+
+            if (!rowContent.data.provided)
+              return false;
```

Comment:
What do you think about https://eslint.org/docs/rules/curly ?

Reply:
I like omitting curlies for one-line statements.

Reply:
Ok, will talk this through while adding more eslint rules (already noted down)